statefulGraph/stateMachineGraph.go

```diff
@@ -76,3 +91,27 @@ func (smg StateMachineGraph) DrawGraph() error {
 
 	return nil
 }
+
+func initializeGraph() (*gographviz.Graph, error) {
```

Comment:
Maybe you rename initializeGraph to intializeGraphWithDir, because that's more obvious then, what the function does

Reply:
I see, it makes sence. I've updated it!